src/main/java/spoon/support/compiler/jdt/JDTCommentBuilder.java

```diff
@@ -373,6 +375,14 @@ public void visitCtAnonymousExecutable(CtAnonymousExecutable e) {
 			public <T> void visitCtParameter(CtParameter<T> e) {
 				e.addComment(comment);
 			}
+
+			@Override
+			public void visitCtCatch(CtCatch e) {
+				if (comment.getPosition().getLine() <= e.getPosition().getLine()) {
```

Comment:
Actually I used the same `getLine` which is used in `visitCtClass` which manage properly multiple inline comments. If I use here `sourceStart()` one comment is missing when writing something like:
```
try {}
// one comment
// two comment
catch {}
```